docs/debugging.md

```diff
@@ -47,9 +47,9 @@ LogBox.ignoreAllLogs();
 
 Unhanded JavaScript errors such as `undefined is not a function` will automatically open a full screen LogBox error with the source of the error. These errors are dismissable and minimizable so that you can see the state of your app when these errors occur, but should always be addressed.
 
-### Syntax Erorrs
+### Syntax Erorr
```

Comment:
Not sure the mistake here was the plural (could be wrong).
That said, "error" would be the right orthograph.